docs-website/docs/overview/sdk.md

```diff
@@ -2,3 +2,92 @@
 sidebar_position: 3
 title: SDK
 ---
+
+# typesafe-i18n
+An opinionated, fully type-safe, lightweight localization library for TypeScript and JavaScript projects with no external dependencies.
```

Comment:
I think it's better to explain why are we using typesafe-i18n instead of copying the tag line.